frb_example/integrate_third_party/rust/src/api/override_web_audio_api.rs

```diff
@@ -103,7 +125,6 @@ handle_audio_node_trait_impls_override!(MediaStreamAudioSourceNode);
 handle_audio_node_trait_impls_override!(MediaStreamTrackAudioSourceNode);
 handle_audio_node_trait_impls_override!(OscillatorNode);
 handle_audio_node_trait_impls_override!(PannerNode);
-handle_audio_node_trait_impls_override!(ScriptProcessorNode);
 handle_audio_node_trait_impls_override!(StereoPannerNode);
```

Comment:
nit: We can keep it not removed, and if Rust complains there is name conflict, we can do something like `#[ext(name=ScriptProcessorNodeSecondExt)]` or whatever.